Add SQL2003 window function syntax

chanmix51 opened the pull request:
Window functions keywords like `WINDOW`, `OVER`, `PARTITION` were not recognized and the SQL formatting was bad when these functions were used.

stof commented:
The fact that you reformat the arrays of keywords/functions make it quite hard to see what is new in them

chanmix51 commented:
@stof true, I have made a diff of before and after:

 * added `WINDOW` after `INTERSECT`
 * added `OVER` after `OUTFILE`

I have tested it in my own project's web debug toolbar and it works well.
